src/Asci/Omnipay/SaferPay/Message/CaptureRequest.php

```diff
@@ -4,14 +4,14 @@
 
 class CaptureRequest extends AbstractRequest
 {
-    protected $endpoint = 'https://www.saferpay.com/hosting/PayCompleteV2.asp';
+    protected $endpoint = 'PayCompleteV2.asp';
 
     public function getData()
     {
         $data = array('ACCOUNTID' => $this->getAccountId(), 'ID' => $this->getTransactionReference());
 
         if ($this->getTestMode()) {
-            $data['spPassword'] = 'XAjc3Kna';
+            $data['spPassword'] = '8e7Yn5yk';
```

Comment:
@ismailasci is this necessary?
It's really hard to spot and it overrides any values that might be set before, a bit misleading...